index.html

```diff
@@ -3383,24 +3388,41 @@ <h3>Verify Proof Sets and Chains</h3>
               </li>
               <li>
 Let |unsecuredDocument| be a copy of |securedDocument| with the proof value
-removed and then set |unsecuredDocument.proof| to |matchingProofs|.
+removed and then set |unsecuredDocument|.|proof| to |matchingProofs|.
               </li>
               <li>
-Run steps 5 through 8 to validate the contents of |proof|.
+Run steps 4 through 8 of the algorithm in section [[[#verify-proof]]] on
+|unsecuredDocument|; if no exceptions are raised, append |cryptosuiteVerificationResult|
+to |verificationResults|.
+                </li>
+            </ol>
+          </li>
+          <li>
+Let |combinedVerificationResult| be an empty struct. Set |combinedVerificationResult|.|verified|
+to `true`, |combinedVerificationResult|.|verifiedDocument| to `null`, and 
+|combinedVerificationResult|.|mediaType| to `null`.
+          </li>
+          <li>
+For each |cryptosuiteVerificationResult| in |verificationResults|:
+            <ol class="algorithm">
+              <li>
+If |cryptosuiteVerificationResult|.|verified| is `false`, set |combinedVerificationResult|.|verified|
+to `false`.
               </li>
               <li>
-Run steps 10 through 15 of the algorithm in section [[[#verify-proof]]]; if no
-exceptions are raised, associate the |isProofVerified| value with this |proof|.
-                <p class="note">
-We specifically use the value of |unsecuredDocument| as set above, rather than
-using the value from step 5 of the algorithm in section [[[#verify-proof]]].
-                </p>
-                </li>
+Otherwise, set |combinedVerificationResult|.|verifiedDocument| to 
+|cryptosuiteVerificationResult|.|verifiedDocument|, and set
+|combinedVerificationResult|.|mediaType| to |cryptosuiteVerificationResult|.|mediaType|.
+              </li>
```

Comment:
```suggestion
Otherwise, set |combinedVerificationResult|.|document| to 
|cryptosuiteVerificationResult|.|verifiedDocument|, set
|combinedVerificationResult|.|mediaType| to |cryptosuiteVerificationResult|.|mediaType|, and
append |cryptosuiteVerificationResult| to |successfulVerificationResults|.
              </li>
```